packages/core/test/StringField.test.jsx

```diff
@@ -3,9 +3,25 @@ import { Simulate } from 'react-dom/test-utils';
 import { fireEvent, act } from '@testing-library/react';
 import sinon from 'sinon';
 import { parseDateString, toDateString, TranslatableString, utcToLocal } from '@rjsf/utils';
+import StringField from '../src/components/fields/StringField';
 
 import { createFormComponent, createSandbox, getSelectedOptionValue, submitForm } from './test_utils';
 
+const StringFieldTest = (props) => {
```

Comment:
Thanks for adding these... Can you also test with a custom Widget inside an string field?